Dockerfile

```diff
@@ -1,80 +1,39 @@
-FROM ubuntu:14.04
-MAINTAINER Danilo Bargen <mail@dbrgn.ch>
-
-# Env variables
-ENV DEBIAN_FRONTEND noninteractive
-
-# Install dependencies
-RUN apt-get update \
-    && sudo apt-get remove libdc1394-22-dev \
-    && apt-get install -y --install-recommends \
-        build-essential \
-                     cmake \
-                     git \
-                     python-pip \
-                     libgdal-dev \
-                     libgeotiff-dev \
-                     pkg-config \
-                     libgtk2.0-dev \
-                     libavcodec-dev \
-                     libavformat-dev \
-                     libswscale-dev \
-                     python-dev \
-                     python-numpy \
-                     libtbb2 \
-                     libtbb-dev \
-                     libjpeg-dev \
-                     libpng-dev \
-                     libtiff-dev \
-                     libjasper-dev \
-                     libflann-dev \
-                     libproj-dev \
-                     libxext-dev \
-                     liblapack-dev \
-                     libeigen3-dev \
-                     libvtk5-dev \
-                     python-networkx \
-                     libgoogle-glog-dev \
-                     libsuitesparse-dev \
-                     libboost-filesystem-dev \
-                     libboost-iostreams-dev \
-                     libboost-regex-dev \
-                     libboost-python-dev \
-                     libboost-date-time-dev \
-                     libboost-thread-dev \
-                     python-empy \
-                     python-nose \
-                     python-pyside \
-                     python-pyexiv2 \
-                     python-scipy \
-                     jhead \
-                     liblas-bin \
-    && apt-get autoremove \
-    && apt-get clean
-
-# Add users
-RUN useradd -m -U odm
+#Pull in previously built packages image with lots of libraries.
+FROM packages
 
 # Prepare directories
 RUN mkdir /code
 WORKDIR /code
 
 # Add repository files
-ADD . /code/
+ADD ccd_defs_check.py /code/ccd_defs_check.py
```

Comment:
What do you think about COPY instead of ADD here? [Docker best practices](https://docs.docker.com/engine/userguide/eng-image/dockerfile_best-practices/#/add-or-copy) recommends it